proto/catalog.proto

```diff
@@ -123,6 +126,8 @@ message Source {
     uint32 associated_table_id = 12;
   }
   string definition = 13;
+
+  // ref connection for connector
   optional uint32 connection_id = 14;
```

Comment:
So we just reuse `Source.connection_id`?

Reply:
Yes and same for the schema registry part and sink.